cmd/zed/permission.go

```diff
@@ -80,6 +104,19 @@ func parseSubject(s string) (namespace, id, relation string, err error) {
 	return
 }
 
+func parseType(s string) (namespace, relation string, err error) {
+	if !strings.Contains(s, "#") {
+		return s, "", nil
+	}
+
+	err = stringz.SplitExact(s, "#", &namespace, &relation)
+	if err != nil {
+		relation = ""
+		err = nil
+	}
+	return
```

Comment:
Because it's only splitting into two things, you're probably better off using [strings.Cut](https://pkg.go.dev/strings#Cut) here. I think by default it's behavior is exactly what we want, so we can even drop the error return variable.

```go
func parseType(s string) (namespace, relation string) {
	namespace, relation, _ = strings.Cut(s, "#")
    return
}
```

Reply:
Neat. Changed